### PR TITLE
Toric code: Add readout unfolding classes (#5)

### PR DESCRIPTION
**Pull request by dstrain115**

### CODE REVIEW

File: recirq/toric_code/readout_unfolding.py
Comment (mpharrigan) [inline recirq/toric_code/readout_unfolding.py:25]: I think this note is inappropriate for the ReCirq repository

File: recirq/toric_code/readout_unfolding.py
Comment (mpharrigan) [inline recirq/toric_code/readout_unfolding.py:31]: is this a statement of principles? what is the practical consequence

File: recirq/toric_code/toric_code_plotter.py
Comment (mpharrigan) [inline recirq/toric_code/toric_code_plotter.py:1]: is this supposed to be a part of this PR? Do you need to merge or rebase?

File: recirq/toric_code/readout_unfolding.py
Comment (mpharrigan) [inline recirq/toric_code/readout_unfolding.py:43]: Not necessary to change, but I'd be loathe to not point out that this class could very well have been a free function

**Review by mpharrigan — APPROVED** [review]
lgtm, pending some minor fixes

**Review by mpharrigan — CHANGES_REQUESTED** [review]
actually I'm going to "request changes" until the merge issue is resolved so things don't accidentally get clobbered. will change back to approval once this is merged or rebased

File: recirq/toric_code/readout_unfolding.py
Comment (dstrain115) [reply recirq/toric_code/readout_unfolding.py]: Removed.

File: recirq/toric_code/readout_unfolding.py
Comment (dstrain115) [reply recirq/toric_code/readout_unfolding.py]: Done.

File: recirq/toric_code/readout_unfolding.py
Comment (dstrain115) [reply recirq/toric_code/readout_unfolding.py:28]: I have no idea.  @kjsatz Can you answer?

**Review by mpharrigan — APPROVED** [review]
lgtm. @kjsatz can you speak to the one outstanding comment?

File: recirq/toric_code/readout_unfolding.py
Comment (dstrain115) [reply recirq/toric_code/readout_unfolding.py]: Synced up.  The consistent ordering is needed for compatibility (between differing conventions in pyle and cirq).

**Pull request merged**
